### ISSUE TITLE
A question for train_action.py

### ISSUE THREAD

**Issue by SoliveTech** [Walter0807/MotionBERT#104]
![d82757739d8e58ba28ea82f652a4221](https://github.com/Walter0807/MotionBERT/assets/56882057/ea6b01aa-d85a-4bfb-811a-e7fd852cfb91)
Thanks for your excellent work！
In the picture from  train_action.py, I find that the "idx" is got in the for loop. but after the loop The if statement calls the  variable "idx " again. Why the "idx "  can be called in here?

**Comment by Walter0807** [comment]
Good catch! You can move it into the loop (pull request is welcome if you want).
